runtime/doc/autocmd.txt

```diff
@@ -553,6 +554,9 @@ CursorMoved			After the cursor was moved in Normal or Visual
 CursorMovedI			After the cursor was moved in Insert mode.
 				Not triggered when the popup menu is visible.
 				Otherwise the same as CursorMoved.
+							*CursorMovedT*
+CursorMovedT			After the cursor was moved in Terminal mode.
+				Equivalent to CursorMovedI.
```

Comment:
Hmm, well as the code stands it will trigger when a popup is visible, I'm still trying to get a popup to in the terminal mode (no idea how vim popups work).  Maybe my documentation is prematurely  written.